docs/quickstart-mission.md

```
@@ -0,0 +1,348 @@
+## Step 3: Search and Select Sensors
+
+Search the sensor catalog, then bind selections to the mission.
+
+### Search for sensors
+
+```bash
+branes sensor search "stereo camera for VIO"
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)